### DISCUSSION TITLE
Pages build failure (Jekyll)<title>

### DISCUSSION THREAD

**Discussion by tterbyenrab** [community/community#40652] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I strongly suspect that something is wrong with the software that GitHub uses to build Pages, as both I and another project manager are now unable to successfully publish projects that we have had no problems with in the recent past. The error log seems to suggest that the problem is a missing file (possibly    /github/workspace/docs
/usr/local/bundle/gems/jekyll-sass-converter-1.5.2/lib/jekyll/converters/scss.rb   ?).

**Comment by yoannchaudet** [comment]
Hello and welcome here, do you have a repo link you can share here? It is hard to investigate or send you in the right direction without.

**Comment by DSDmark** [comment]
Hi @tterbyenrab,

Try to change this inside the in page section:-

![image](https://user-images.githubusercontent.com/85744816/205428146-9f970b1c-5872-4835-aba3-a270b9dcf815.png)

if its not work then,
And Don't deploy your site on  **Jekyll-page**. just use simple **static HTML**.

![image](https://user-images.githubusercontent.com/85744816/205428201-22749dd2-24d2-4d62-bdea-2ccdf9849ab5.png)

Here is quick link:- [**setting**](https://github.com/DSDmark/TodoList/settings/pages)

**Reply by tterbyenrab** [reply]
Thank you *very* much, @DSDmark. I am such a novice here that I don't even know how to describe what I'm seeing, so I especially appreciate your kind help.
